cpg-language-python/src/test/resources/python/datatypes.py

```diff
@@ -7,4 +7,10 @@
     "e": "f"
 }
 e = f'Values of a: {a} and b: {b!s}'
-f = a[1:3:2]
\ No newline at end of file
+f = a[1:3:2]
```

Comment:
Please have a separate test code file specific for this test. Reusing this datatypes.py should be avoided.